astropy/utils/tests/test_data.py

```diff
@@ -903,6 +904,17 @@ def test_find_invalid():
         )
 
 
+def test_get_invalid():
```

Comment:
To avoid repetition, can `pytest.mark.parametrize` this test to accept `package=None`, `package='astropy'`, and `package='numpy'`. Numpy is to test "external" package call, as you have asked, and should be a safe bet since it is required dependency.

Reply:
Thanks!